src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -82,3 +84,36 @@ def upload_file(
         print(f"{datetime.now()}: {msg}")
         logger.info(msg)
         return None
+
+    def kick_off_ingest(self) -> None:
```

Comment:
nitpick: not that it matters, but maybe `start_ingest` or `trigger_ingest_process` would be less metaphorical?